### PR TITLE
WIP: Force color scheme

### PR DESCRIPTION
**Pull request by joehoyle**
Remove the WordPress color schemes, using our one #everytime. See https://github.com/humanmade/platform-dev/issues/321 as a final decision hasn't been made for this yet.

### CODE REVIEW

File: inc/branding/namespace.php [hunk 1]
```diff
@@ -3,26 +3,39 @@
...
+	</g>
+</svg>
+END;
 /**
```
Comment (rmccue) [inline inc/branding/namespace.php]: Missing newline above this.

File: inc/branding/namespace.php [hunk 2]
```diff
@@ -115,3 +131,36 @@ function detect_missing_default_theme() {
...
+
+	$wp_admin_bar->add_menu( $logo_menu_args );
+}
+
+/**
+ * Remove the WordPress admin color schemes.
+ *
+ * In Platform we have a single color scheme, so we don't need the default WordPress ones,
+ * and we don't need the color scheme picker.
+ *
```
Comment (rmccue) [inline inc/branding/namespace.php]: Extra line here.

File: inc/branding/namespace.php [hunk 3]
```diff
@@ -115,3 +131,36 @@ function detect_missing_default_theme() {
 
 	wp_die( $message, $title, [ 'response' => WP_Http::NOT_FOUND ] );
 }
+
+
+/**
+ * Add the Platform logo menu.
+ *
+ *
```
Comment (rmccue) [inline inc/branding/namespace.php]: Extra line here.

**Comment by roborourke** [comment]
@joehoyle think we can close this one now

**Pull request closed without merging**